backend/typescript/middlewares/validators/adminValidators.ts

```diff
@@ -2,6 +2,7 @@ import { Request, Response, NextFunction } from "express";
 import { validateFormQuestion } from "./formQuestionValidators";
 import { validatePrimitive, getApiValidationError } from "./util";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Disabled whatever eslint `any` checks that have been in the codebase forever, thought it was about time we do so.

Reply:
sg!